Libraries/Microsoft.Teams.AI.Models.OpenAI/OpenAIChatModel.cs

```diff
@@ -29,34 +27,34 @@ public partial class OpenAIChatModel : IChatModel<ChatCompletionOptions>
     /// <summary>
     /// the logger instance
     /// </summary>
-    protected ILogger Logger { get; set; }
+    protected ILogger<OpenAIChatModel> Logger { get; }
 
-    public OpenAIChatModel(string model, OpenAIClient client)
+    public OpenAIChatModel(string model, OpenAIClient client, ILogger<OpenAIChatModel>? logger = null)
     {
         Model = model;
         ChatClient = client.GetChatClient(model);
-        Logger = new ConsoleLogger(model);
+        Logger = logger ?? LoggerFactory.Create(builder => {}).CreateLogger<OpenAIChatModel>();
```

Comment:
I would avoid creating new loggers, instead what we can do is to use the NullLogger.Instance as the default value